tensorflow/lite/examples/label_image/label_image.h

```diff
@@ -17,25 +17,33 @@ limitations under the License.
 #define TENSORFLOW_LITE_EXAMPLES_LABEL_IMAGE_LABEL_IMAGE_H_
 
 #include "tensorflow/lite/string.h"
+#if defined(ANDROID) || defined(__ANDROID__)
+#include "tensorflow/lite/delegates/gpu/gl_delegate.h"
```

Comment:
Can this import be removed?

Reply:
`#include "tensorflow/lite/delegates/gpu/gl_delegate.h"`? I guess not, need it for something like 'TfLiteGpuDelegateOptions'